src/clients/SpokePoolClient.ts

```diff
@@ -110,21 +111,33 @@ export class SpokePoolClient extends BaseAbstractClient {
     this.configStoreClient = hubPoolClient?.configStoreClient;
   }
 
+  public eventSignatures: Record<string, string> = {
+    FundsDeposited: "FundsDeposited(bytes32,bytes32,uint256,uint256,uint256,uint256,uint32,uint32,uint32,bytes32,bytes32,bytes32,bytes)",
+    FilledRelay: "FilledRelay(bytes32,bytes32,uint256,uint256,uint256,uint256,uint256,uint32,uint32,bytes32,bytes32,bytes32,bytes32,bytes32,(bytes32,bytes32,uint256,uint8))",
+    RequestedSpeedUpDeposit: "RequestedSpeedUpDeposit(uint256,uint256,bytes32,bytes32,bytes,bytes)",
+    RequestedSlowFill: "RequestedSlowFill(bytes32,bytes32,uint256,uint256,uint256,uint256,uint32,uint32,bytes32,bytes32,bytes32,bytes32)",
+  };
```

Comment:
This is temporary and should be deleted once a new contracts release is available.